assets/xml/overlays/ovl_Magic_Wind.xml

```diff
@@ -0,0 +1,15 @@
+<Root>
+    <File Name="ovl_Magic_Wind" BaseAddress="0x80B8B100" RangeStart="0x780" RangeEnd="0x1C98" Segment="128">
+        <Skeleton Name="sdSkel" Type="Curve" LimbType="Curve" Offset="0x1C90"/>
```

Comment:
there's a `d` that you probably didn't mean to put in the symbol names in this xml